CHANGES.txt

```diff
@@ -4,14 +4,33 @@ Changelog
 2.1.0 (unreleased)
 ------------------
 
+Announcements:
+
+* Change all references to the pep8 project to say pycodestyle; #530
+
 Changes:
 
-* Added check E74x for using variables named 'l', 'O', or 'I'; #341
-* Improved performance of `compound_statements` check; #314 / #522
-* Fixed remaining references to `pep8`; #518 / #530
-* Added `noqa` support for `maximum_line_length` check; #538
-* Added check E305 for two blank lines after toplevel methods and classes; #400
+* Add Gitter badge; #557
+* Report E302 for blank lines before an "async def"; #556
+* Updated tox to do same tests as Travis CI (`setup.py test`).
+* Updated to use sheilds.io badges.
+* Updated text about running tests with tox; #552
+* Update maximum_line_length to use Checker.noqa; 538
+* Updated Travis CI doc link.
+* Updated docs related to tested Python versions.
```

Comment:
Perhaps "Update our list of tested and supported Python versions which are now ..."